lib/narn-lib.js

```diff
@@ -22,45 +39,69 @@ exports.detectYarn = function detectYarn(args) {
 
 exports.getYarnArgs = narnArgs => narnArgs;
 
-exports.getNpmArgs = narnArgs => {
-  const yarnArgs = minimist(narnArgs, { boolean: ["dev", "D"] });
+exports.getNpmArgs = (narnArgs = [], isPnpm = false) => {
+  const yarnArgs = minimist(narnArgs, {
+    boolean: ["dev", "D"],
+    alias: {
+      D: "dev"
+    }
+  });
   const yarnCommands = yarnArgs._;
   const yarnTarget = yarnCommands.length > 0 ? yarnCommands[0] : "install";
   const yarnSubCommands = yarnCommands.slice(1);
+  const narnSubArgs = narnArgs.slice(1);
+
   let npmTarget;
   let npmArgs;
   let result;
   switch (yarnTarget) {
     case "install":
       npmTarget = "install";
-      npmArgs = [];
+      npmArgs = narnSubArgs;
       break;
     case "add":
       npmTarget = "install";
       const packages = yarnSubCommands.map(transformNarnPackageString);
-      const saveType = yarnArgs.dev || yarnArgs.D ? "--save-dev" : "--save";
+      const saveType = yarnArgs.dev ? "--save-dev" : "--save";
```

Comment:
I just added the alias in minimist, which makes this expression cleaner

Reply:
👍